### PR TITLE
MWPW-130474: remove placeholder syntax from price links

### PR DESCRIPTION
**Pull request by Axelcureno**
Removes placeholder syntax from price links generated by OST

Resolves: [MWPW-130474](https://jira.corp.adobe.com/browse/MWPW-130474)

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/?martech=off
- After: https://mwpw-130474--milo--adobecom.hlx.page/?martech=off

### CODE REVIEW

**Review by yesil — APPROVED** [review]

**Review by mokimo — APPROVED** [review]
Please fix the failing OST unit test before merging, otherwise looks good 👍

File: libs/blocks/ost/ost.css [hunk 1]
```diff
@@ -0,0 +1 @@
+/* created due of block nature in Milo */
```
Comment (honstar) [inline libs/blocks/ost/ost.css:1]: FWIW, I started #752 to discuss this requirement.

Comment (Axelcureno) [reply libs/blocks/ost/ost.css:1]: Since the conversation is still ongoing may I suggest we leave this for now for the sake of removing the 404?

Comment (Axelcureno) [reply libs/blocks/ost/ost.css]: File is no longer empty :)

**Pull request merged**